.readthedocs.yml

```
@@ -1,12 +1,14 @@
 version: 2
 
-python:
-  install:
-  - method: pip
-    path: .
-    extra_requirements:
-      - docs
-
-mkdocs:
-  configuration: mkdocs.yml
-  fail_on_warning: false
\ No newline at end of file
+build:
+  os: ubuntu-lts-latest
+  tools:
+    # `mambaforge-latest` will now start failing: we just need a working `mamba`
+    python: mambaforge-23.11
+  commands:
+    - git fetch --tags
```

```suggestion
    - git fetch --tags --unshallow
```

the shallow clone makes it impossible to get the tags for the version. using an unshallow fetch might fix that.

https://github.com/readthedocs/readthedocs.org/issues/5989

---

the other suggestion would be 

```suggestion
    - git fetch --tags --depth=1000
```